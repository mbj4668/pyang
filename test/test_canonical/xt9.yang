module xt9 {
  yang-version 1.1;
  ext:foo bar;
  prefix "xt9";
  namespace "urn:uri:foo";   // expect namespace before prefix

  import ext { prefix ext; }
  contact "hej" { }
  description "arne" + /* hmm*/ ' ' + 'anka';
  organization "hopp";  // expected after import

  revision 2001-01-01 { description "2"; }
  revision 2000-01-11 { description "initinla"; }

  typedef myInt2 {
    status deprecated;  // expected after type
    type int8 {
      range "1 | 5..6";
    }
  }
  leaf ss {
    type string {
      pattern "hej";
      length 3; // expected before pattern
    }
  }
}
