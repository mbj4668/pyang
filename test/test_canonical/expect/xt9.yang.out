xt9.yang:4: error: UNEXPECTED_KEYWORD_CANONICAL_1_v1.1
xt9.yang:5: error: UNEXPECTED_KEYWORD_CANONICAL_v1.1
xt9.yang:10: error: UNEXPECTED_KEYWORD_CANONICAL_v1.1
xt9.yang:16: error: UNEXPECTED_KEYWORD_CANONICAL_1_v1.1
xt9.yang:17: error: UNEXPECTED_KEYWORD_CANONICAL_v1.1
xt9.yang:24: error: UNEXPECTED_KEYWORD_CANONICAL_v1.1
